lib/teaspoon/server.rb

```diff
@@ -4,9 +4,10 @@
 
 module Teaspoon
   class Server
-    attr_accessor :port
+    attr_accessor :port, :host
 
     def initialize
+      @host = Teaspoon.configuration.server_host || '127.0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.